bdio-protobuf/src/main/proto/bdba_scan_nodes_v1.proto

```diff
@@ -0,0 +1,89 @@
+syntax = "proto3";
+
+package bdio.v1;
+
+import "google/protobuf/timestamp.proto";
+
+option java_multiple_files = true;
+option java_package = "com.blackducksoftware.bdio.proto.v1";
+
+
+enum MatchType {
+  SOME_MATCH_TYPE = 0;
+}
+
+
+/**
+ * See here for guidelines on avoiding breaking changes to message formats: https://developers.google.com/protocol-buffers/docs/proto#updating
+ * If those are not possible, or the message is sufficiently different, a new version should be created.
+ */
+message ProtoDependencyNode {
+  // the numeric id of component node
+  int64 depends_on = 1;
+
+  // the numeric id of file node
+  optional int64 evidence = 2;
```

Comment:
Similar comment in other protos.   Add documentation that describes why optional fields are optional.

Reply:
I will have meeting with BDBA team on this week, and we are going to go over each field and determine if it's required or optional, etc. After that I'll update documentation.